template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseCloudStorageServiceConfigProvider.java

```diff
@@ -54,13 +54,13 @@ public boolean isConfigurationNeeded(CmTemplateProcessor cmTemplateProcessor, Te
                 .map(SharedServiceConfigsView::isDatalakeCluster)
                 .orElse(false);
         String cdhVersion = getCdhVersion(source);
-        boolean is722OrNewer = isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERAMANAGER_VERSION_7_2_2);
+        boolean is727OrNewer = isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERAMANAGER_VERSION_7_2_7);
         String accountId = ThreadBasedUserCrnProvider.getAccountId();
         boolean sdxHbaseCloudStorageEnabled =
                 entitlementService.sdxHbaseCloudStorageEnabled(accountId);
         return source.getFileSystemConfigurationView().isPresent()
                 && cmTemplateProcessor.isRoleTypePresentInService(getServiceType(), getRoleTypes())
-                && (!datalakeCluster || (is722OrNewer && sdxHbaseCloudStorageEnabled));
+                && (!datalakeCluster || (is727OrNewer && sdxHbaseCloudStorageEnabled));
```

Comment:
Nit: What about UT? If there is no associated test class, it might make sense to add one now.

Reply:
UT is updated.